hosts/ace.nix

```diff
@@ -17,4 +17,8 @@
 
   boot.loader.efi.canTouchEfiVariables = lib.mkForce true;
   boot.loader.systemd-boot.enable = true;
+
+  environment.systemPackages = [
+    self.packages.${pkgs.hostPlatform.system}.clang-morello
```

Comment:
Mhm. I would rather make your flake depended on the doctor cluster flake configuration. Pulling in build tools into the global environment has it's problems. Also does this actually works with the normal glibc anyway.

Reply:
It did work, I could compile stuff with this clang. But adding it to the dev flake is better, thanks.